test/expected/wiki.out

```diff
@@ -1,34 +1,23 @@
-CREATE EXTENSION IF NOT EXISTS vector;
-CREATE EXTENSION
-CREATE EXTENSION IF NOT EXISTS lanterndb;
-CREATE EXTENSION
+\qecho
 
+\set ON_ERROR_STOP on
 SET statement_timeout = 0;
-SET
 SET lock_timeout = 0;
-SET
 SET idle_in_transaction_session_timeout = 0;
-SET
 SET client_encoding = 'UTF8';
```

Comment:
The SQL corresponding to all these `SET` logs is unnecessary and does not exist elsewhere.
originally, `wiki.sql` was the result of a `pg_dump`, that is how all of this ended up here. But We should delete all that (now or later) so the wiki test looks like the rest of the tests.

Reply:
I will modify the test in this PR